frontend.go

```diff
@@ -137,17 +137,19 @@ func Frontend(ctx context.Context, c client.Client) (*client.Result, error) {
 		return nil, err
 	}
 
-	st, err := cg.Generate(ctx, call, file)
+	_, err = cg.Generate(ctx, mod, []*parser.CallStmt{call})
```

Comment:
I dont understand this code, why are we calling Generate and then throwing away the solve.Request? What is the point?

Reply:
I didn't want to spend too much time on `frontend.go` because its deprecated. It used to be a way to `hlb publish` a function, but we use libraries for that now. Maybe I should just delete this file.